unitex/src/fr/umlv/unitex/leximir/delas/ChooseDelas.java

```diff
@@ -166,7 +166,7 @@ public boolean accept(File dir, String name) {
                     }
                 };
 
-                if (allDFoler.list(filter).length == 0) {
+                if (allDFolder.list(filter).length == 0) {
                     JOptionPane.showMessageDialog(null, "No Delas was Found in : " + DictionaryPath.allDelas, "Information", JOptionPane.INFORMATION_MESSAGE);
```

Comment:
What about `No DELAS file was found in ` 

- DELAS is an acronym,
- It is not necessary to add a non-breaking space before the colon.

Reply:
Changes done!